app/models/user.rb

```diff
@@ -241,9 +241,14 @@ def can_super_admin?
   #
   # Returns Boolean
   def can_org_admin?
-    self.can_grant_permissions? || self.can_modify_guidance? ||
-      self.can_modify_templates? || self.can_modify_org_details? ||
-      self.can_review_plans?
+    return true if can_super_admin?
```

Comment:
I think that this makes sense on the nose, but maybe calls for more thought about our role structure.

I like the idea of a super-admin implicitly having org_admin permissions, but I wonder how many other places in the code we check for the granular permission, rather than the categorical (i.e. org/super_admin).